server.js

```diff
@@ -1,9 +1,11 @@
 // BASE SETUP
 var express = require('express');
 var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
I like how this is higher up the stack, but do we want to look into a way of taking the port out of this file? Maybe to some global var seeing as I've noticed different members are using different ports on testing. And testing in the future may involve this port changing.